aiida/storage/sqlite_temp/backend.py

```diff
@@ -120,12 +127,13 @@ def get_session(self) -> Session:
             self._session.commit()
         return self._session
 
-    def get_repository(self) -> SandboxRepositoryBackend:
+    def get_repository(self) -> SandboxShaRepositoryBackend:
         if self._closed:
             raise ClosedStorage(str(self))
         if self._repo is None:
             # to-do this does not seem to be removing the folder on garbage collection?
-            self._repo = SandboxRepositoryBackend(filepath=get_config_option('storage.sandbox') or None)
+            repo_path = self.profile.storage_config.get('repo_path')
```

Comment:
The above to-do should be incorrect? At least the `SandboxRepositoryBackend` implements the `__del__` method to delete the folder on disk.

Reply:
yeh, I recall observing some times when, on exiting an ipython session, this did not appear to "activate", but I can give it another try...